lib/components/narrative/default/default-itinerary.js

```diff
@@ -148,12 +143,36 @@ const ITINERARY_ATTRIBUTES = [
         // FIXME: For CAR mode, walk time considers driving time.
         <>
           <FormattedDuration duration={itinerary.walkTime} />
-          <LegIconWrapper>
+          <LegIconWrapper noSpace>
```

Comment:
Why do we need the noSpace? The icon looks the same as the others

Reply:
There was a margin around LegIconWrapper and I wasn't sure why, so instead of removing it I just added this prop.